bokeh/models/annotations.py

```diff
@@ -1096,6 +1097,44 @@ class Whisker(Annotation):
 
     level = Override(default="underlay")
 
+class InfoPane(Annotation):
+    ''' Renders a info-pane.
+
+    '''
+    level = Override(default="overlay")
+
+    # TODO: can be modified to be anchored at corners as well
+    anchor = Enum(TooltipAttachment, help="""
+    Whether the pane should be anchored at the left or right of the pane
+    or above or below it, or if it should be automatically placed
+    in the horizontal or vertical dimension.
+    """)
+
+    x = Either(Auto, NumberSpec(), default=0, help="""
```

Comment:
Another change in https://github.com/bokeh/bokeh/pull/10237 that we may want to consider mirroring here is the combination of separate x and y properties into a single `position` property. The advantage of that is that updating the position only requires changing one property value instead of two. Considering our event driven system, updating two either results in extra work being done (since both x and y changes trigger events), or requires special hoops to jump through to avoid doing the extra work.